tests/ibpt-v1.test.js

```diff
@@ -0,0 +1,217 @@
+const axios = require('axios');
+
+const validTestArray = expect.arrayContaining([
+  expect.objectContaining({
+    codigo: expect.any(String),
+    ex: expect.any(String),
+    tipo: expect.any(String),
+    descricao: expect.any(String),
+    nacionalfederal: expect.any(Number),
+    importadosfederal: expect.any(Number),
+    estadual: expect.any(Number),
+    municipal: expect.any(Number),
+    vigenciainicio: expect.any(String),
+    vigenciafim: expect.any(String),
+    chave: expect.any(String),
+    versao: expect.any(String),
+    fonte: expect.any(String),
+  }),
+]);
+
+const validTestObject = expect.objectContaining({
+  codigo: expect.any(String),
+  ex: expect.any(String),
+  tipo: expect.any(String),
+  descricao: expect.any(String),
+  nacionalfederal: expect.any(Number),
+  importadosfederal: expect.any(Number),
+  estadual: expect.any(Number),
+  municipal: expect.any(Number),
+  vigenciainicio: expect.any(String),
+  vigenciafim: expect.any(String),
+  chave: expect.any(String),
+  versao: expect.any(String),
+  fonte: expect.any(String),
```

Comment:
Acho que podemos colocar alguns `_` para que fiquem mais legíveis
```suggestion
    codigo: expect.any(String),
    ex: expect.any(String),
    tipo: expect.any(String),
    descricao: expect.any(String),
    nacional_federal: expect.any(Number),
    importados_federal: expect.any(Number),
    estadual: expect.any(Number),
    municipal: expect.any(Number),
    inicio_vigencia: expect.any(String),
    fim_vigencia: expect.any(String),
    chave: expect.any(String),
    versao: expect.any(String),
    fonte: expect.any(String),
  }),
]);

const validTestObject = expect.objectContaining({
  codigo: expect.any(String),
  ex: expect.any(String),
  tipo: expect.any(String),
  descricao: expect.any(String),
  nacional_federal: expect.any(Number),
  importados_federal: expect.any(Number),
  estadual: expect.any(Number),
  municipal: expect.any(Number),
  inicio_vigencia: expect.any(String),
  fim_vigencia: expect.any(String),
  chave: expect.any(String),
  versao: expect.any(String),
  fonte: expect.any(String),
```